src/test/test-log.c

```diff
@@ -70,6 +70,54 @@ static void test_log_syntax(void) {
         assert_se(log_syntax("unit", LOG_ERR, "filename", 10, SYNTHETIC_ERRNO(ENOTTY), "ENOTTY: %s: %m", "hogehoge") == -ENOTTY);
 }
 
+static void test_log_context(void) {
+        char *strv[] = { (char*) "MYDATA=abc", NULL };
```

Comment:
STRV_MAKE?

maybe add multiple entries here, to actually use the fact an strv is accepted here